data.sql

```diff
@@ -120,4 +120,86 @@ UPDATE animals
 SET owner_id = (SELECT owner_id FROM owners WHERE full_name = 'Dean Winchester')
 WHERE name IN ('Angemon', 'Boarmon');
 
+---------------------------------------------------------------------------------------------
+
+-- Insert the following data for vets:
+-- Vet William Tatcher is 45 years old and graduated Apr 23rd, 2000.
+-- Vet Maisy Smith is 26 years old and graduated Jan 17th, 2019.
+-- Vet Stephanie Mendez is 64 years old and graduated May 4th, 1981.
+-- Vet Jack Harkness is 38 years old and graduated Jun 8th, 2008.
+
+INSERT INTO vets (name, age, date_of_graduation)
+VALUES ('William Tatcher', 45, '2000-04-23'),
+       ('Maisy Smith', 26, '2019-01-17'),
+       ('Stephanie Mendez', 64, '1981-05-04'),
+       ('Jack Harkness', 38, '2008-06-08');
+
+---------------------------------------------------------------------------------------------
+
+-- Insert the following data for specialties:
+-- Vet William Tatcher is specialized in Pokemon.
+-- Vet Stephanie Mendez is specialized in Digimon and Pokemon.
+-- Vet Jack Harkness is specialized in Digimon.
+
+INSERT INTO specializations (vet_id, species_id)
+VALUES (1, 2),
+       (3, 1),
+       (3, 2),
+       (4, 1);
```

Comment:
- [ ] [OPTIONAL] Instead of adding the associations by `id` manually which is prone to mistakes, you can dynamically retrieve the `ids` like so 👇🏾 

```sql
 INSERT INTO specializations (species_id, vet_id)
 VALUES ((SELECT id FROM vets WHERE name = 'William Tatcher'), (SELECT id FROM species WHERE name = 'Pokemon'));

 INSERT INTO specializations (species_id, vet_id)
 VALUES ((SELECT id FROM vets WHERE name = 'Stephanie Mendez'), (SELECT id FROM species WHERE name = 'Pokemon'));

 INSERT INTO specializations (species_id, vet_id)
 VALUES ((SELECT id FROM vets WHERE name = 'Stephanie Mendez'), (SELECT id FROM species WHERE name = 'Digimon'));

 INSERT INTO specializations (species_id, vet_id)
 VALUES ((SELECT id FROM vets WHERE name = 'Jack Harkness'), (SELECT id FROM species WHERE name = 'Digimon'));
```